### PR TITLE
✨ #165 add basic login page

### LINKED ISSUE #165: Login page
Right now the login page only consists of a loginbtn. There should be created a design in Figma and the page should subsequently be implemented.

- [x] create login pg design in Figma
- [x] implement login pg

### PR DESCRIPTION
**Pull request by VigdisIS**
Adds a login page instead of just a button:

image

Closes #165

### CODE REVIEW

**Comment by Terbau** [comment]
I'm having issues with the image disappearing quickly after load.
vv

**Comment by VigdisIS** [comment]
> I'm having issues with the image disappearing quickly after load. vv [ vv ](https://user-images.githubusercontent.com/40792825/199345617-1c104cb0-3088-4cea-900d-d3b4858ca504.gif) [ ](https://user-images.githubusercontent.com/40792825/199345617-1c104cb0-3088-4cea-900d-d3b4858ca504.gif)

Seems to be a bug fixed by adding the SVG code in the file instead of a img with the SVG as src 🤔

**Review by Terbau — APPROVED** [review]
LGTM :rocket:

File: frontend/src/components/Login.svelte [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+            style="fill:rgb(44,44,44);fill-rule:nonzero;"></path>
+        </g>
+      </g>
+    </svg>
```
Comment (annaiandresen) [inline frontend/src/components/Login.svelte:51]: just a thought for later, but might be a bit cleaner to put svgs in assets. think you can import them like a svelte component!

File: frontend/src/components/footer/FooterGithub.svelte [hunk 2]
```diff
@@ -1,9 +1,8 @@
 <div class="footer_github">
-  Rapportere feil på siden? Legg inn issue på
-  <a class="hover:underline" href="https://github.com/dotkom/vengeful-vineyard">
-    Github</a
+  Forslag til features?🏷️ Feil på siden?🐛 Legg inn issue på
+  <a class="underline" href="https://github.com/dotkom/vengeful-vineyard">
+    Github.</a
   >
-  og gi vinstraff til Joakim!
```
Comment (annaiandresen) [inline frontend/src/components/footer/FooterGithub.svelte:6]: rip :(

**Review by annaiandresen — APPROVED** [review]
lgtm!!

File: frontend/src/components/Login.svelte (hunk 1 above)
Comment (VigdisIS) [reply frontend/src/components/Login.svelte:51]: Ref Brage's comment https://github.com/dotkom/vengeful-vineyard/pull/168#issuecomment-1299249195, seems to be fixed by adding the SVG directly in the file:o. Should probably be another cleaner solution out there though.

File: frontend/src/components/footer/FooterGithub.svelte (hunk 2 above)
Comment (VigdisIS) [reply frontend/src/components/footer/FooterGithub.svelte:6]: ✝️

**Pull request merged**